### ISSUE TITLE
Is there any 'live filter' and calculate band power in realtime module?

### ISSUE THREAD

**Issue by iPsych** [mne-tools/mne-realtime#18]
I am very confused about how to combine mne and mne-realtime.

Is there any 'live filter' and calculate band power in realtime module?

**Comment by jasmainak** [comment]
no, you only have data coming in real time. The computations should be made reasonably fast to stay real time.

any contributions to making it better are very welcome
